modules/serverbidBidAdapter.js

```diff
@@ -125,84 +79,114 @@ ServerBidAdapter = function ServerBidAdapter() {
       if (placement.networkId && placement.siteId) {
         data.placements.push(placement);
       }
-    }
+    });
 
-    if (data.placements.length) {
-      ajax(config.BASE_URI, _responseCallback, JSON.stringify(data), { method: 'POST', withCredentials: true, contentType: 'application/json' });
-    }
-  }
+    ret.data = JSON.stringify(data);
+    ret.bidRequest = validBidRequests;
+    ret.url = ENDPOINT_URL;
 
-  function _responseCallback(result) {
+    return ret;
+  },
+
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse, bidRequest) {
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like
```JavaScript
serverResponse = serverResponse.body;
```
just below this line, or however you'd prefer to grab the `body`, and updating corresponding tests if needed should get this back to working properly